.claude/agents/onboarding.md

```
@@ -383,12 +383,23 @@ Running: ./scripts/project setup
 
 [Show output from the command]
 
-✅ Virtual environment created!
+✅ Setup complete!
+```
 
-To activate it, run:
-  source .venv/bin/activate
+**IMPORTANT**: After the setup command finishes, remind the user to activate the virtual environment:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)